mesheryctl/internal/cli/root/mesh/mesh.go

```diff
@@ -21,11 +21,15 @@ var MeshCmd = &cobra.Command{
 // Lifecycle management of service meshes
 mesheryctl mesh 
 	`,
-	Args: cobra.MinimumNArgs(1),
+
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New(utils.MeshError(fmt.Sprintln("requires at least 1 arg(s), only received 0 ")))
+		}
 		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
-			return errors.New(utils.SystemError(fmt.Sprintf("invalid command: \"%s\"", args[0])))
+			return errors.New(utils.MeshError(fmt.Sprintf(" '%s' command not found. See 'mesheryctl mesh --help'\n ", args[0])))
```

Comment:
Similarly...
`[___] command not found. Use 'mesheryctl mesh --help' for usage guide`

Reply:
Ok , I will do that 😄